src/test/java/org/folio/rest/impl/TransactionsSummariesTest.java

```diff
@@ -0,0 +1,25 @@
+package org.folio.rest.impl;
+
+import java.net.MalformedURLException;
+
+import org.folio.rest.jaxrs.model.OrderTransactionSummary;
+import org.junit.jupiter.api.Test;
+
+import io.vertx.core.json.JsonObject;
+
+class TransactionsSummariesTest extends TestBase {
+
+  private static final String ORDER_TRANSACTION_SUMMARIES_ENDPOINT = "/finance-storage/order-transaction-summaries";
+  private static final String ORDER_TRANSACTION_SUMMARIES_ENDPOINT_WITH_ID = ORDER_TRANSACTION_SUMMARIES_ENDPOINT + "/{id}";
+
+  @Test
+  void testOrderTransactionSummaries() throws MalformedURLException {
+    OrderTransactionSummary sample = new JsonObject(getFile("transaction_summary.sample")).mapTo(OrderTransactionSummary.class);
```

Comment:
can you please rename the sample file to be more evident to orders, as we will also have invoice transaction summary